chromium/popup.html

```diff
@@ -19,13 +19,13 @@
     <h1 data-i18n="about_ext_name"></h1>
 </header>
 
-<section id="disableButton" class="options">
+<section id="disableButton" class="options" style="display:none;">
     <div class="onoffswitch">
         <input id="onoffswitch" type="checkbox" checked>
         <label data-i18n="menu_globalEnable" for="onoffswitch"></label>
     </div>
 </section>
-<section id="HttpNowhere" class="options">
+<section id="HttpNowhere" class="options" style="display:none;">
```

Comment:
No need of a semicolon here.

Reply:
As above, would you mind motivating this? In future reviews, if a change is stylistic, or if it is a case where both ways are correct, would you mind motivating your comments?

I usually find that small stylistic changes like this can be a time sink, with little benefit.